static/js/components/ShiftPage.jsx

```diff
@@ -124,6 +130,9 @@ const ShiftPage = ({ route }) => {
               <CurrentShiftMenu currentShift={currentShift} />
             ) : null)}
         </Paper>
+        <Paper elevation={1}>
+          <CommentOnShift />
+        </Paper>
```

Comment:
```suggestion
        <Paper elevation={1}>
        {(shiftList && !show && currentShift ? (
          <CommentOnShift />
          ) : null)}
        </Paper>
```

Reply:
Also above for the `<CurrentShiftMenu/>`, in the if statement the `currentShift` is duplicated, could you also remove that duplicate? Thanks.
You can simply copy and paste the if statement given here in the suggestion.